CHANGELOG.md

```diff
@@ -5,7 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.15.0] - 2023-09-28
+
+### Added
+
+### Changed
+
+- `Container::exec` changed to be synchronous and return `ExecOutput`
+- MSRV is now 1.63.
+
+### Removed
+
+- Removed all pre-defined images from the library to escape unbounded maintenance work.
+  See https://github.com/testcontainers/testcontainers-rs/issues/471 for details.
+- Removed explicit support for podman.
+  See https://github.com/testcontainers/testcontainers-rs/issues/422 for details.
```

Comment:
Are these really the only things we want to have mentioned in the changelog?

Reply:
It seems like we forgot to update the changelog for several PRs, I've quickly went through the list here and I think we should mention the following ones:

- https://github.com/testcontainers/testcontainers-rs/pull/384
- https://github.com/testcontainers/testcontainers-rs/pull/410
- https://github.com/testcontainers/testcontainers-rs/pull/496
- https://github.com/testcontainers/testcontainers-rs/pull/437
- https://github.com/testcontainers/testcontainers-rs/pull/383
- https://github.com/testcontainers/testcontainers-rs/pull/394

Reply:
Left comment bellow with suggestion of missed ones 🙂